tracer/src/Datadog.Trace/RemoteConfigurationManagement/ApplyDetails.cs

```diff
@@ -0,0 +1,34 @@
+// <copyright file="ApplyDetails.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
```

Comment:
Please use `#nullable enable` on all new files

Reply:
fixed in 8bdccfd